### PR TITLE
FIXED #7284 - Top of dashlets being cut off by nav bar nd positioning of dashlet pop-up

### LINKED ISSUE #7284: Top of dashlets being cut off by nav bar
#### Issue

The top of dashlets are being cutoff by the navbar on chrome. This only happens on specific display settings with any dashlet except the first dashlet at the top of the page. This occurs in `MySugar.js` in the `configureDashlet` function. The if statement around line 203 calculates if a newTop is needed and will calculate the top (in my case) to -90px which cuts off the dashlet. 

This does not happen on Firefox.

dashletCutoff

#### Expected Behavior

The dashlet should not be cut off

#### Actual Behavior

The dashlet gets cut off

#### Possible Fix

My quick fix was to just subtract 50 from the newTop calculation. I don't know what an actual fix should look like though. 

#### Steps to Reproduce

1. Windows 10 display settings > Advanced scalling settings: This must be set to 125%
2. Screen resolution: 1920 x 1080
3. Open chrome to the dashboard page
4. The first dashboard will open fine, all other dashboards will open with the top cut off by the navbar

#### Your Environment

* SuiteCRM Version used: 7.10.2
* SuiteCRM Theme used: SuiteP
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome version 74
* Environment name and version (e.g. MySQL, PHP 7):
* Operating System and version (e.g Ubuntu 16.04): windows 10

### PR DESCRIPTION
**Pull request by cameronblaikie**
## Description

The top of dashlets are being cutoff by the navbar on chrome. 

## Motivation and Context

https://github.com/salesagility/SuiteCRM/issues/7284
## How To Test This

1. go to home page
2. edit dashlet on home page
3. scroll up and down to make sure dashlet doesn't disappear behind the nav bar.

Other fix instructions below.

1. Add lots or random dashlets.
2. scroll to the bottom of the home page.
3. click edit on the very bottom dashlet of home page.
4. make sure the dashlet opens in the middle of the screen from the position you are on the screen and DOESN'T throw you back to the top of the screen.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: themes/SuiteP/css/suitep-base/editview.scss [hunk 1]
```diff
@@ -5429,6 +5429,7 @@ body.yui-skin-sam.masked .yui-panel-container.yui-dialog.yui-simple-dialog.shado
   top: 100px;
   padding: 20px;
   width: 100%;
+  position: fixed;
   z-index: 103100;
```
Comment (Dillon-Brown) [inline themes/SuiteP/css/suitep-base/editview.scss]: Is the ```z-index: 103100``` really necessary here?

File: jssource/src_files/include/MySugar/javascript/MySugar.js [hunk 2]
```diff
@@ -198,11 +198,11 @@ initMySugar = function () {
           SUGAR.util.evalScript(result['body']);
 
           // calculate the scroll and dashlet popup positions
-          var rlTop = 200;
-          var newTop = $("#dashlet_" + dashletId).offset().top - rlTop;
-          if (newTop + $('#dlg').outerHeight(true) > $('#dlg_mask').height()) {
-            newTop -= (newTop + $('#dlg').outerHeight(true) - $('#dlg_mask').height() + rlTop);
-          }
+          // var rlTop = 50;
```
Comment (Dillon-Brown) [inline jssource/src_files/include/MySugar/javascript/MySugar.js]: Commented out code should be removed.

**Comment by gunnicom** [comment]
Did you look into https://github.com/salesagility/SuiteCRM/issues/5695 , too?

**Comment by Mac-Rae** [comment]
Assessed :+1:

**Pull request merged**